coderd/prebuilds/api.go

```diff
@@ -9,7 +9,10 @@ import (
 	"github.com/coder/coder/v2/coderd/database"
 )
 
-var ErrNoClaimablePrebuiltWorkspaces = xerrors.New("no claimable prebuilt workspaces found")
+var (
+	ErrNoClaimablePrebuiltWorkspaces = xerrors.New("no claimable prebuilt workspaces found")
+	ErrAGPLDoesNotSupportPrebuilds   = xerrors.New("prebuild-related functionality is not supported under the AGPL license")
```

Comment:
Let's keep the terminology consistent ("prebuilt workspaces")

Reply:
done